### PR TITLE
Update

### PR DESCRIPTION
**Pull request by weizmm**

### CODE REVIEW

File: src/util/updateSpecificFields.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+const matchToKartoffel = require('./matchToKartoffel');
+const axios = require('axios');
+const p = require('../config/paths');
+const logger = require('./logger');
+
+/**
+ * This module accept an array that contain DeepDiff objects and build from them object for the PUT request that send to Kartoffel
+ * @param {*} deepDiffArray Array of DeepDiff objects 
+ * @param {*} dataSource The source of the raw person object
```
Comment (Daniel231) [inline src/util/updateSpecificFields.js]: add person obj to documtation

File: src/util/updateSpecificFields.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+ */
+const updateSpecificFields = (deepDiffArray, dataSource, person) => {
+    deepDiffArray.map(async (deepDiffRecord) => {
+        let objForUpdate = {};
+        switch (deepDiffRecord.kind) {
+            case "E":
+                console.log("E");
```
Comment (Daniel231) [inline src/util/updateSpecificFields.js]: remove

File: src/util/updateSpecificFields.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+                objForUpdate[deepDiffRecord.path.toString()] = deepDiffRecord.rhs;
+                break;
+            case "N":
+                console.log("N");
```
Comment (Daniel231) [inline src/util/updateSpecificFields.js]: remove

File: src/util/updateSpecificFields.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+/**
+ * This module accept an array that contain DeepDiff objects and build from them object for the PUT request that send to Kartoffel
+ * @param {*} deepDiffArray Array of DeepDiff objects 
+ * @param {*} dataSource The source of the raw person object
+ */
+const updateSpecificFields = (deepDiffArray, dataSource, person) => {
+    deepDiffArray.map(async (deepDiffRecord) => {
+        let objForUpdate = {};
+        switch (deepDiffRecord.kind) {
```
Comment (Daniel231) [inline src/util/updateSpecificFields.js]: change to if condition

File: src/util/diffsHandler.js [hunk 5]
```diff
@@ -60,10 +84,47 @@ require('dotenv').config();
...
+                })
+                if (deepDiffForUpdate.length > 0) {
+                    updateSpecificFields(deepDiffForUpdate, dataSource, person.data);
+                    await domainUserHandler(person.data, record[1], true, dataSource);
+                };
+            }
+            else {
+                // Add secondary domain user from the record (if the required data exist)
+                await domainUserHandler(person.data, record[1], false, dataSource);
+                // !!!NEED TO CHANGE THE LOG IF THE FIELD THAT UPDATED IS DOMAINUSER!!!
```
Comment (Daniel231) [inline src/util/diffsHandler.js]: !!!NEED TO CHANGE THE LOG IF THE FIELD THAT UPDATED IS DOMAINUSER!!!

File: src/util/updateSpecificFields.js (hunk 1 above)
Comment (weizmm) [reply src/util/updateSpecificFields.js]: resolve

File: src/util/updateSpecificFields.js (hunk 2 above)
Comment (weizmm) [reply src/util/updateSpecificFields.js]: resolve

File: src/util/updateSpecificFields.js (hunk 3 above)
Comment (weizmm) [reply src/util/updateSpecificFields.js]: resolve

File: src/util/updateSpecificFields.js (hunk 4 above)
Comment (weizmm) [reply src/util/updateSpecificFields.js]: resolve

**Pull request merged**